modules/phase_field/src/userobjects/DiscreteNucleationInserter.C

```diff
@@ -79,6 +79,7 @@ DiscreteNucleationInserter::threadJoin(const UserObject &y)
   // combine _local_nucleus_list entries from all threads on the current process
   const DiscreteNucleationInserter & uo = static_cast<const DiscreteNucleationInserter &>(y);
   _local_nucleus_list.insert(_local_nucleus_list.end(), uo._local_nucleus_list.begin(), uo._local_nucleus_list.end());
+  _changes_made += uo._changes_made;
```

Comment:
Ok, got it, this is only called on thread 0 with the UOs for thread 1..N passed in!